apps/meteor/client/components/message/variants/RoomMessage.tsx

```diff
@@ -41,6 +41,7 @@ const RoomMessage = ({ message, sequential, all, mention, unread, context, ignor
 	const chat = useChat();
 	const messageRef = useRef(null);
 
+	const showUserAvatar = useUserPreference<boolean>('displayAvatars');
```

Comment:
would you mind passing that value via prop?
I'm afraid this hook might add a bit of burden to rendering as it's the same for all list items, maybe you could use it in MessageList.tsx

Reply:
Sure @ggazzo

Reply:
Please check now.